PIasynAxis compiler warnings about initialization problems

These warnings occur during the ``vxWorks-ppc32-debug`` build:

```
../PIasynAxis.h: In constructor 'PIasynAxis::PIasynAxis(PIasynController*, PIGCSController*, int, const char*)':
../PIasynAxis.h:73: warning: 'PIasynAxis::m_bMoving' will be initialized after
../PIasynAxis.h:68: warning:   'bool PIasynAxis::m_bEnableAxisAfterHoming'
../PIasynAxis.cpp:43: warning:   when initialized here
```

--- a/pigcs2App/src/PIInterface.cpp
+++ b/pigcs2App/src/PIInterface.cpp
@@ -61,7 +61,7 @@ asynStatus PIInterface::sendOnly(const char *outputBuff, asynUser* logSink)
     if (status != asynSuccess)
     {
         asynPrint(logSink, ASYN_TRACE_ERROR|ASYN_TRACEIO_DRIVER,
-                  "PIGCSController:sendOnly: error sending command %s, sent=%d, status=%d\n",
+                  "PIGCSController:sendOnly: error sending command %s, sent=%zu, status=%d\n",
                   outputBuff, nActual, status);
     }
     return(status);
@@ -83,7 +83,7 @@ asynStatus PIInterface::sendOnly(char c, asynUser* logSink)
     if (status != asynSuccess)
     {
         asynPrint(logSink, ASYN_TRACE_ERROR|ASYN_TRACEIO_DRIVER,
-                  "PIGCSController:sendOnly: error sending command %d, sent=%d, status=%d\n",
+                  "PIGCSController:sendOnly: error sending command %d, sent=%zu, status=%d\n",
                   int(c), nActual, status);
         //printf("PIInterface::sendOnly() sending \"#%d\"\n", int(c));
     }

--- a/pigcs2App/src/PIasynAxis.h
+++ b/pigcs2App/src/PIasynAxis.h
@@ -65,12 +65,12 @@ public:
     asynUser* m_pasynUser;
 
     bool m_bHasLimitSwitches;
-    bool m_bEnableAxisAfterHoming;
-    bool m_bSetServoAfterHoming;
     bool m_bHasReference;
     bool m_bProblem;
     bool m_bServoControl;
     bool m_bMoving;
+    bool m_bEnableAxisAfterHoming;
+    bool m_bSetServoAfterHoming;
     int m_movingStateMask;
     u_PIGCS2PiezoCLParams m_CloseLoopParam;
 

